content/es/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -0,0 +1,57 @@
+---
+reviewers:
+- raelga
+title: Pipeline de métricas de recursos
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+Desde Kubernetes 1.8, las métricas de uso de recursos, tales como el uso de CPU y memoria del contenedor,
+están disponibles en Kubernetes a través de la Metrics API. Estas métricas pueden ser o accedidas directamente
+por el usuario, por ejemplo usando el comando `kubectl top`, o usadas por un controlador en el cluster,
+por ejemplo el Horizontal Pod Autoscaler, para la toma de decisiones.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## La Metrics API
+
+A través de la Metrics API, puedes obtener la cantidad de recursos usados
+actualmente por cada nodo o pod. Esta API no almacena los valores de las métricas,
+así que no es posible, por ejemplo, obtener la cantidad de recursos usados por
+un nodo dado hace 10 minutos.
+
+La API no es diferente a otra API:
```

Comment:
La traducción `is not different from` quizás es demasiado directa, en castellano es más común `La API de métricas está completamente integrada en la API de Kubernetes:`